irr/src/CSkinnedMesh.cpp

```diff
@@ -511,7 +511,7 @@ void CSkinnedMesh::skinJoint(SJoint *joint, SJoint *parentJoint)
 			jointVertexPull.transformVect(thisVertexMove, weight.StaticPos);
 
 			if (AnimateNormals)
-				jointVertexPull.rotateVect(thisNormalMove, weight.StaticNormal);
+				jointVertexPull.rotateAndScaleVect(thisNormalMove = weight.StaticNormal);
```

Comment:
Eww. Please don't use the assignment operator like this, it's so ugly and visually hides the assignment.

Reply:
Yeah, could as well use comma operator at this point... or just an if-block.

Reply:
I considered on the verge of unreadable, but also somewhat neat ;)